lstchain/tests/test_lstchain.py

```diff
@@ -130,6 +130,9 @@ def test_r0_to_dl1_lhfit_mc(tmp_path, mc_gamma_testfile):
     config['lh_fit_config']["use_interleaved"] = True
     config['waveform_nsb_tuning']['nsb_tuning'] = True
     r0_to_dl1(mc_gamma_testfile, custom_config=config, output_filename=tmp_path / "tmp.h5")
+    os.remove(tmp_path / "tmp.h5")
+    config['waveform_nsb_tuning']['pre_computed_multiplicity'] = 0
```

Comment:
It defaults to 10 from line 432 in r0_to_dl1.py (and in the class, but this is not used) and is not defined in the standard config file.
I should probably add it there too as being 10 for consistency. (Or make it None in the default config file and change the code to support it)
And in this test it may be clearer if I explicitly select its value as 10 to avoid confusion.